### ISSUE TITLE
Container not jumping to pages beyond 3 (Even after visiting page 5)

### ISSUE THREAD

**Issue by Twisted-Code** [Wynntils/Wynntils#2859]
[This time](https://github.com/Wynntils/Wynntils/issues/2795) it's not moving at all (unless the page index is 3 or less), and my inventory is not full.
I have a hypothesis (since it was working a few minutes prior to the issue) that it is because I just unlocked a page in my personal bank. Do you keep track of the personal bank jump pages separately? If not, it might think my account bank (which has more pages) is as big as my personal bank. Although if this is the case I'm not sure why it will go up to 3 (my personal bank has only the second page unlocked, I don't have VIP).
Where is that file and JSON key I need to check to discover where it thinks the last page is?

**Comment by Twisted-Code** [comment]
Okay I checked the JSON data, it's right and I see that you have separate indexes stored. I have no other hypotheses so I will wait for a reply.
Edit: that said it did seem to start working again after visiting the final page... weird, maybe I'm on to something after all.

**Comment by ShadowCat117** [comment]
Have recently added some logging for quick jumps and also made the buttons text colour change depending on if Wynntils thinks they are accessible so if you are able to replicate it again, those logs would be useful

**Comment by Twisted-Code** [comment]
Yeah I noticed the button colors, figured that was what that meant. I'll let you know
